src/test/isolation2/sql/resgroup/resgroup_dumpinfo.sql

```diff
@@ -99,4 +99,3 @@ SELECT count(*) FROM pg_resgroup_get_status_kv(NULL);
 DROP ROLE role_dumpinfo_test;
 DROP ROLE role_permission;
 DROP RESOURCE GROUP rg_dumpinfo_test;
-DROP LANGUAGE plpython3u CASCADE;
```

Comment:
Let's remove `CREATE LANGUAGE plpython3u;` in the top of this file. Because it confuse when we create the language but don't drop it.